treeherder/model/derived/jobs.py

```diff
@@ -363,15 +363,20 @@ def get_max_job_id(self):
 
     @staticmethod
     def get_performance_series_cache_key(project, interval_seconds,
-                                         hash=False):
-        key = 'performance-series-summary-%s-%s' % (project,
-                                                    interval_seconds)
+                                         machine_platform=None, hash=False):
+        if machine_platform is None:
+            key = 'performance-series-summary-%s-%s' % (project,
+                                                        interval_seconds)
+        else:
+            key = 'performance-series-summary-%s-%s-%s' % (project,
+                                                           interval_seconds,
+                                                           machine_platform)
         if hash:
             key += '-hash'
 
         return key
 
-    def get_performance_series_summary(self, interval_seconds):
+    def get_performance_series_summary(self, parameter):
```

Comment:
I don't think consolidating interval_seconds and machine_platform into a single variable is a good idea here (that's more of a javascript thing to do). So I'd write function signature as `get_performance_series_summary(self, interval_seconds, machine_platform=None)`